test_scripts/rbac-scram-provided-debian-upgrade-550-current.sh

```diff
@@ -10,9 +10,6 @@ export START_UPGRADE_VERSION=5.5
 
 export CURRENT_VERSION=true
 
-## If upgrading from 6.2.0 or later, this should be false as upgrade playbooks no longer exist.  Upgrades are handled via reconfiguration.
-export UPGRADE_PLAYBOOK=true
```

Comment:
I didn't get purpose of these changes. Ideally we should not touch / change these scripts.

Reply:
I was removing an unused variable. If the script is generalised, this variable will not be used anywhere.
This variable was required to manually state if upgrading from 6.2.0 or later. We can resolve that info from the start branch.